src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -1600,6 +1600,99 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
     return {{element::f32, element::f32}, {element::i32, element::i32}};
 }
 
+/// Tanh ///
+jit_tanh_emitter::jit_tanh_emitter(ov::intel_cpu::riscv64::jit_generator* host,
+                                   ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                   const std::shared_ptr<ov::Node>& /*node*/,
+                                   ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc),
+      jit_exp_emitter_(std::make_unique<jit_exp_emitter>(host, host_isa, exec_prc)) {
+    prepare_table();
+}
+
+jit_tanh_emitter::jit_tanh_emitter(ov::intel_cpu::riscv64::jit_generator* host,
+                                   ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                   ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc),
+      jit_exp_emitter_(std::make_unique<jit_exp_emitter>(host, host_isa, exec_prc)) {
+    prepare_table();
+}
+
+size_t jit_tanh_emitter::get_inputs_num() const {
+    return 1;
+}
+
+size_t jit_tanh_emitter::aux_gprs_count() const {
+    OPENVINO_ASSERT(jit_exp_emitter_, "JIT Exp emitter is missed!");
```

Comment:
Could you please replace `OPENVINO_ASSERT` with `OV_CPU_JIT_EMITTER_ASSERT`? This assert is more informative in bounds of `jit_emitter`

Reply:
This comment is still valid - please double-check